### ISSUE TITLE
Error in testing room database

### ISSUE THREAD

**Issue by nimishph** [udacity/andfun-kotlin-sleep-tracker#27]
in app/build.gradle if the following line

        testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'

is not there under buildConfig then we get the following error

> junit.framework.AssertionFailedError: No tests found in com.example.android.trackmysleepquality.SleepDatabaseTest

**Comment by tessarakt** [comment]
I don't think this is completed. I needed to add the testInstrumentationRunner in defaultConfig **in addition to** the changes in the linked comment.
